City-of-Helsinki/yjdh
# HL-68 | Upgrade HDS to the latest version

## Description :sparkles:
- I made [a fix in HDS](https://github.com/City-of-Helsinki/helsinki-design-system/pull/500) for the DatePicker component to accept empty values, and the fix is released in version 1.13.1

## Testing :alembic:
- Go to the first step of the application
- Under "Hakijan de minimis-tuet", choose "Kyllä"
- Fill in the inputs of then click "Lisää"
- The date picker in the new row should have an empty value
